modules/build/src/main/scala/scala/build/Inputs.scala

```diff
@@ -404,6 +436,7 @@ object Inputs {
             List(resolve(url, content))
         }
       }
+      else if (arg.endsWith("project.settings.scala")) Right(Seq(ProjectSettings(dir, subPath)))
```

Comment:
Not sure about the `arg.endsWith` here. Wouldn't this work better?
```suggestion
      else if (path.last == "project.settings.scala") Right(Seq(ProjectSettings(dir, subPath)))
```